Sources/pixelpass/pixelpass.swift

```diff
@@ -99,27 +107,39 @@ public class PixelPass {
         return nil
     }
     
-    public func getMappedCborData(jsonData: [String:String], mapper: [String:String]) -> [UInt8] {
-        var payload = [String: String]()
-        for (param, value) in jsonData {
-            let key = mapper[param] ?? param
-            payload[key] = value
-        }
-        let cborEncodableData = convertToCBOREncodableFormat(input: payload)
-        return cborEncodableData.encode()
+    public func getMappedData(stringData: String, mapper: [String:String], cborEnable : Bool = false) -> String {
+        let jsonData = stringData.data(using: .utf8)!
+        let mappedJSON = translateToJSON(jsonData: jsonData, mapper: mapper)
+        do {
+            if !cborEnable{
+                let decoded =  try JSONSerialization.data(withJSONObject: mappedJSON, options: [])
+                if let str = String(data: decoded, encoding: .utf8) {
+                    return str
+                }
+            }
+        }catch {}
```

Comment:
should we throw any error here?

Reply:
We can. we can throw a JSON error maybe. will update and push.

Reply:
Updated